### PR TITLE
[PF-1906] RBS Readme updates

### PR DESCRIPTION
**Pull request by dexamundsen**
Documentation updates

https://broadworkbench.atlassian.net/browse/PF-1906

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +66,9 @@ In order to use a runtime pool configuration, the environment variable `BUFFER_P
   * For existing pools, pool sizes may change, but resource configurations may not.
   * If a previously created pool does not exist under `BUFFER_POOL_SYSTEM_FILE_PATH`, it will be deleted.
 
-## Development
+## Development, testing & deployment
+[Github interactions](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/DEVELOPMENT.md#github-interactions)
```
Comment (zloery) [inline README.md:70]: Is this the right link? RBS has a different set of github actions workflows than WSM

File: README.md [hunk 2]
```diff
@@ -114,26 +133,23 @@ To use this, first ensure Skaffold is installed on your local machine
...
- credentials with `gcloud auth configure-docker`. Finally, run local-run.sh with
-  your target environment as the first argument:
-
+You may need to use gcloud to provide GCR credentials with `gcloud auth configure-docker`.
+Alternately use `gcloud auth login <you>@broadinstitute.org` or `gcloud auth application-default login` for fine grain control of the login credentials to be used (account must have permissions to push to GKE)
 ```
-cd local/dev
+cd local-dev
 ```
+You may also use your [personal environment](https://github.com/DataBiosphere/terra/blob/main/docs/dev-guides/personal-environments.md) as the target
```
Comment (zloery) [inline README.md:141]: actually, I think the personal environments are the only environment people will have permission to target

**Review by zloery — APPROVED** [review]
Thanks for the refresh on this doc!

**Review by yonghaoy — APPROVED** [review]

**Comment by yonghaoy** [comment]
LG. FedRamp may require associating Jira issue to PR.  If Gary asks for a JIRA ticket next week, you can create one https://broadworkbench.atlassian.net/browse/PF-82, and update PR title.

File: README.md (hunk 1 above)
Comment (dexamundsen) [reply README.md:70]: Noticed it when working on the JDK update PR, removed the link and replaced it with a placeholder

File: README.md (hunk 2 above)
Comment (dexamundsen) [reply README.md:141]: Updated

File: README.md (hunk 1 above)
Comment (dexamundsen) [reply README.md:70]: Updated

**Pull request merged**